test/unit/dialects/mssql.js

```diff
@@ -30,4 +30,26 @@ describe('MSSQL unit tests', () => {
       'select * from [projects] where [id" = 1 UNION SELECT 1, @@version -- --] = ?'
     );
   });
+
+  it('should include "?" in statement', async () => {
```

Comment:
test naming is pretty cryptic, maybe following would be more clear about tests intent:

`should not convert escaped "?" -marks in raw SQL to native parameter bindings` 

the same applies to second test:

`should convert "?" -marks in raw SQL to native parameter binding`